snowflake/database/migrations/000001_create_flows_table.up.sql

```diff
@@ -47,5 +47,7 @@ CREATE TABLE flows (
   reverseThroughputFromSourceNode NUMBER(20, 0),
   reverseThroughputFromDestinationNode NUMBER(20, 0),
   clusterUUID STRING(36),
-  timeInserted TIMESTAMP_TZ DEFAULT current_timestamp()
+  timeInserted TIMESTAMP_TZ DEFAULT current_timestamp(),
+  egressName STRING(256),
```

Comment:
Do `egressName` and `egressIP` need to come before `timeInserted` in order to match the S3 csv data format? I'm not sure about whether the snowpipe copy statement will work correctly in this case or not. Have you tried that?

Reply:
They come after timeInserted actually, yes I have verified a snowflake setup manually